docs/source/models/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -1023,21 +1023,24 @@ Code is below for reference
     - No new issues found with adding correlation. `Causes with correlation <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Cause_VV_correlation_07.25.2023.ipynb>`_ and `risks with correlation <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/risks_VV_07.25.2023.ipynb>`_ were checked in the same manner as previous runs. Additionally, `simulant exposure values <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/interactive_correlation.ipynb>`_ were checked for correlation in the interactive sim both on initialization and after 25 time steps. It was found that correlation varied around the expected input values. 
   * - 17.0
-    - Add New Observer 
-    - 
+    - Add New Observer - Binned Exposure  
+    - The new observer seems to be working well, V&V for this observer only can be found in the `binned exposure notebook <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/binned_risks_08.08.2023.ipynb>`_. 
   * - 18.0
     - Test Runs for Timing on 10 States 
-    - States to be included are: Alabama, Washington, Alaska, Rhode Island, California, Texas, Kansas, West Virginia, Florida, North Dakota  
+    - These runs were completed and the V&V was saved to this `folder <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/tree/main/10_locations_7.18.23>`_. V&V largely worked well and no state specific issues were noted. There was an issue with randomness being included between scenarios which was resolved and fixed.
   * - 19.0
     - Adjusting Theraputic Inertia 
-    - 
+    - After adjustments to theraputic inertia, initialized values for medication are more stable throughout the sim. This workbook included the `latest medication rates <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Risks_VV_08_3_23.ipynb>`_. There are still three outstanding questions: (1) the no treatment group increases over time in the sim  for both medications (2) very few simulants move to higher treatment categories and (3) the age for LDL-C medication change should be lower in men to prevent drift. However, we are leaving this until after the 51 location run is working to reassess. 
   * - 20.0
-    - Add Medication and PAF Recalculations 
-    - 
+    - Single draw run in 51 locations 
+    - Runs were completed with draw 829. The V&V for causes and risks looked to be expected in this notebook for `51 location causes <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Cause_VV_multi_location_8.4.23.ipynb>`_ and `51 location risks <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Risks_VV_multi_location_8_7_23.ipynb>`_. However, there were issues with overlap between scenarios. We are still investigating this but it might be related to heart failure. Additionally, there are still issues with the PAFs containing zero values. We are currently avoiding this by selecting specific draws, however, a more permanent solution would be preferred. 
```

Comment:
Should I be concerned about how different the blue and orange are in these plots?

![image](https://github.com/ihmeuw/vivarium_research/assets/51236/34342cd5-ea7b-4eec-83a1-ecfb1f0b7eea)

Reply:
And why are the stroke plots in cell [57] so different than the validation plots above?

Reply:
Something funny goes on with the cell - I think I am reusing a name accidentally. I have fixed it before and it is a notebook issue not a data/sim issue but you're right, I should make sure it is fixed permanently!

Reply:
Got a little merge happy but I did fix this. The formula wasn't updated to include scenarios properly which made it generate weird results. 

![image](https://github.com/ihmeuw/vivarium_research/assets/108029462/19ff90c9-36b0-46f0-890b-d94d7bb26895)